apps/src/applab/designMode.js

```diff
@@ -309,6 +309,38 @@ designMode.updateProperty = function(element, name, value) {
     case 'readonly':
       element.setAttribute('contenteditable', !value);
       break;
+    case 'data-is-default':
```

Comment:
I would call the property itself is-default rather than data-is-default. The data portion is only needed for the attribute in the DOM (vs. the property name which is something we just use internally)